libraries/rush-lib/src/cli/actions/ChangeAction.ts

```diff
@@ -690,4 +702,23 @@ export class ChangeAction extends BaseRushAction {
   private _logNoChangeFileRequired(): void {
     console.log('No changes were detected to relevant packages on this branch. Nothing to do.');
   }
+
+  private _stageGitChanges(patterns: string[]): void {
+    try {
+      patterns.forEach((pattern) => {
+        child_process.execSync(`git add "${pattern}"`);
```

Comment:
Instead of this you can use Rush shared helper command executor from [here](https://github.com/microsoft/rushstack/blob/c7aef037ee57dbb0892b2882c9d8404cc51ce0fc/libraries/rush-lib/src/utilities/Utilities.ts#L319)

Reply:
Changed to that